WordPress/src/main/res/values/strings.xml

```diff
@@ -1061,6 +1061,14 @@
     <string name="scan_history">Scan History</string>
     <string name="scan_history_menu_title">History</string>
 
+    <string name="scan_loading_title">Loading scan</string>
+    <string name="scan_no_network_title" translatable="false">@string/no_network_title</string>
+    <string name="scan_no_network_subtitle" translatable="false">@string/no_network_message</string>
+    <string name="scan_request_failed_title" translatable="false">Something went wrong</string>
```

Comment:
⚠️ It seems this string uses "translatable=false" attribute, even though it contains a translation and not just a reference. Wdyt?

Reply:
Good catch. After I added it, I searched for "Something went wrong" in the `strings.xml `as I remembered I saw it earlier. But it wasn't the same and had an extra string added to it, then I forgot to remove "translatable=false" 🤦‍♀️.

In cf7dfd6.